docs/settings/fleet-settings.asciidoc

```diff
@@ -41,9 +51,100 @@ See the {fleet-guide}/index.html[{fleet}] docs for more information.
   | Hostnames used by {agent} for accessing {fleet-server}.
 | `xpack.fleet.agents.elasticsearch.hosts`
   | Hostnames used by {agent} for accessing {es}.
+| `xpack.fleet.agents.elasticsearch.ca_sha256`
+  | Hash pin used for certificate verification. The pin is a base64-encoded
+    string of the SHA-256 fingerprint.
 |===
 
-[NOTE]
-====
-In {ecloud}, {fleet} flags are already configured.
-====
+
+//QUESTION: Is the xpack.fleet.agents.fleet_server.hosts setting supported? It
+//wasn't listed in https://github.com/elastic/observability-docs/issues/758.
+
+==== Preconfiguration settings
+
+//QUESTION: We should add some details here to help users understand
+//when they might need to modify these settings? Are these customizations that
+//most users won't need to know about?
```

Comment:
these settings let users pre-define integrations and agent policies they want Fleet to load up by default, so that they do not have to manually do them in the UI. we added these to enable Cloud to configure Cloud-specific policies. IMO this is an advanced case that most users won't use